controllers/keystoneapi_controller.go

```diff
@@ -908,6 +914,11 @@ func (r *KeystoneAPIReconciler) reconcileNormal(
 			err.Error()))
 		return ctrlResult, err
 	} else if (ctrlResult != ctrl.Result{}) {
+		instance.Status.Conditions.Set(condition.FalseCondition(
+			condition.TLSInputReadyCondition,
+			condition.RequestedReason,
+			condition.SeverityInfo,
+			fmt.Sprintf(condition.TLSInputReadyWaitingMessage, "one or more cert secrets")))
```

Comment:
@stuggi I wanted to get your input here.  The call to `ValidateCertSecrets` on L907 no longer returns an error with the name of the missing TLS `Secret` because we (I) updated `lib-common` to not return an error when we're also returning a `RequeueAfter` [1].  So here we don't know which TLS cert `Secret` is missing, hence we're using `"one or more cert secrets"` as the requeue status message.  Just wondering if you think we should refactor the chain from `ValidateCertSecrets` through `VerifySecret` in `lib-common` to be able to return a missing `Secret` name, so that we can be more explicit here.

[1] https://github.com/openstack-k8s-operators/lib-common/pull/547

Reply:
we get the RequeueAfter only if the secrets are really missing. if there is an error with the content we get an error, right? So we could either do what you said, or we just have a generic message like ""TLSInput is missing: %s", where %s is  "%s and/or %s", `instance.Spec.TLS.API.Internal.SecretName`, `instance.Spec.TLS.API.Public.SecretName`"